indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
I don't find half full-hd (960x1080) qualifies as tiny screen particularly since Windows  [allows to arrange two windows in such a way easily](http://blogs.msdn.com/b/e7/archive/2009/03/17/designing-aero-snap.aspx).

Moreover people don't always run their applications full-screen either because they have something else aside or they simply don't like to read long lines.

I'll look for other places where this negative margin was applied and will be happy to fix any glitch properly.